tracer/tracer.go

```diff
@@ -1165,6 +1077,7 @@ func (t *Tracer) probabilisticProfile(interval time.Duration, threshold uint) {
 	enableSampling := false
 	var probProfilingStatus = probProfilingDisable
 
+	//nolint:gosec
```

Comment:
This is to suppress lint warning on using non-secure random generator. This is ok here as this just for probability and not security related.